torchgeo/datasets/vhr10.py

```diff
@@ -0,0 +1,544 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""NWPU VHR-10 dataset."""
+
+import os
+from typing import Any, Callable, Dict, List, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from matplotlib import patches
+from PIL import Image
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import check_integrity, download_and_extract_archive, download_url
+
+
+def convert_coco_poly_to_mask(
+    segmentations: List[int], height: int, width: int
+) -> Tensor:
+    """Convert coco polygons to mask tensor.
+
+    Args:
+        segmentations (List[int]): polygon coordinates
+        height (int): image height
+        width (int): image width
+
+    Returns:
+        Tensor: Mask tensor
+    """
+    from pycocotools import mask as coco_mask  # noqa: F401
+
+    masks = []
+    for polygons in segmentations:
+        rles = coco_mask.frPyObjects(polygons, height, width)
+        mask = coco_mask.decode(rles)
+        # if len(mask.shape) < 3:
+        #     mask = mask[..., None]
+        mask = torch.as_tensor(mask, dtype=torch.uint8)
+        mask = mask.any(dim=2)
+        masks.append(mask)
+    if masks:
+        masks_tensor = torch.stack(masks, dim=0)
+    else:
+        masks_tensor = torch.zeros((0, height, width), dtype=torch.uint8)
+    return masks_tensor
+
+
+class ConvertCocoAnnotations:
+    """Callable for converting the boxes, masks and labels into tensors.
+
+    This is a modified version of ConvertCocoPolysToMask() from torchvision found in
+    https://github.com/pytorch/vision/blob/main/references/detection/coco_utils.py
```

Comment:
```suggestion
    https://github.com/pytorch/vision/blob/v0.14.0/references/detection/coco_utils.py
```
This URL will be more stable in case the feature is ever removed